assets/src/application/ui/layout/entityList/pagination/usePaginationConfig.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Same as the file above :point_up:

Reply:
Also we might extract `onChange` logic here from pagination if it will became complex.